core/sections/clause_8_vector.adoc

```diff
@@ -18,25 +19,44 @@
 
 * color: Color
 
-==== Class Stroke
-===== Class GraphicalUnit
-===== Class StrokeStyling
+==== Class Graphic
 
-==== Class MultiGraphic
-===== Class Marker
+* opacity: float
+* transform: Transform2D
 
-==== Class Graphic
 ===== Class Image
-====== Class Resource
-===== Class Text
-The text property will exceptionnally support an identifier parameter value expression (without any additional conformance class required).
 
-====== Class Font
-====== Class TextAlignment
+* image: Resource
+* hotSpot: Pointf
+* tint: Color
+* blackTint: Color
+* alphaThreshold: float
 
-* enum HAlignment
-* enum VAlignment
+===== Class Text
+
+* text: string
+* font: Font
+* alignment: TextAlignment
 
 ===== Class Shape
+
 ====== Class Dot
-* point: Pointf
+
+===== Class MultiGraphic
+
+* elements: Graphic [0..*]
+
+====== Class Marker
+
+==== Class Resource
+
+==== Class Pointf    
+==== Class GraphicalUnit
+==== Class StrokeStyling
+==== Class Stroke
```

Comment:
We are missing the Stroke properties here (but only the fact that it inherits from StrokeStyling with a color and width property / graphical unit).
We want to explain that Marker inherits from MultiGraphic.

Pointf is in the context of Dot point property, so we can describe under that section.